interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -1,8 +1,8 @@
 <div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
   {{$ctrl.errorMessage}}
 </div>
-<select
-  ng-options="option as option.title for option in $ctrl.availableMlModels"
-  ng-model="$ctrl.selectedMlModel">
-  <option value="">--- Please select a machine learning model ---</option>
+<select name="selectMlModel"
+  ng-options="opt.title for opt in $ctrl.availableMlModels track by opt.id"
```

Comment:
I found that the select box would not properly respond to an already-selected mlmodel without this detail.